src/pages/Login.tsx

```diff
@@ -28,10 +29,13 @@ export const Login: React.FC<RouteComponentProps> = ({ location }) => {
     cb(e.target.value);
   };
 
+  if (isProduction()) {
```

Comment:
I think this should go after the isAuthenticated condition

Reply:
good point. done.